### ISSUE TITLE
Documentation needs wrok

### ISSUE THREAD

**Issue by TRusselo** [home-assistant/home-assistant.github.io#21713]
### Feedback

needs to include networking/dns resolving requirements for non .io users connecting to a separate host.

### URL

https://www.home-assistant.io/integrations/pi_hole/

### Version

2022.2.9

**Comment by frenck** [comment]
I'm not entirely sure what you mean or try to suggest.
Could you extend your suggestion to make it more clear?

**Comment by TRusselo** [comment]
honestly, i dont know. i cant get this to work with the insturcitons provided, with searching,  other people who has the issue, someone else mentions DNS resolving and the issue is resolved somehow.
i dont know.
https://github.com/home-assistant/core/issues/61788

**Comment by frenck** [comment]
I think you are mixing up things, looking at the issue you linked.

<img width="380" alt="CleanShot 2022-02-21 at 03 36 31@2x" src="https://user-images.githubusercontent.com/195327/154879928-a48363ea-ae10-420c-aedf-b19f4743fc5c.png">

Port 53, is the port that runs DNS in Pi-Hole. You are not configuring DNS servers in this case.

**Comment by cliffordwhansen** [comment]
@TRusselo The PiHole details you are entering here are for the PIHole API, which by default runs on port 80 (443 if you have SSL configured)

PIHole also uses port 53 but that is for the DNS requests and not used by this integration.

Looking at the issue you linked, you said you using unraid, it looks like the container is not exposing the web ports, if you expand the arrow in the bottom right corner do you see other ports?

**Comment by TRusselo** [comment]
@cliffordwhansen @frenck 
tired 80 first...  spent 4 hours trying to get this working.

cliff - i think you are on the right track... but i dont know~~!

![image](https://user-images.githubusercontent.com/1154815/154996233-49944b3a-e9c7-4aad-92e9-4da54ff39806.png)
you mean this?

**Comment by TRusselo** [comment]
dont know what im supposed to do with this information....

step 1 install pihole
step 2 install integration
fail.

**Comment by cliffordwhansen** [comment]
What url do you access PIHole on? I'm guessing http://192.168.1.20?
And what is the IP of your HomeAssistant server?

It might be that they cannot see each other on the network?

**Comment by TRusselo** [comment]
yes http://192.168.1.20 is pihole running in a docker on br0 (bridge 0?) on my urnraid server
and http://192.168.1.12 is the unraid server
and http://192.168.1.12:8123 is home assistant docker as a host on the same unraid server
i can browse all from  .14 .16 ,ect, any ip on the same subnet

**Comment by cliffordwhansen** [comment]
Since they are running on the save docker let try something.

When setting up PIHole in HomeAssistant, try using `pihole` as the hostname and `80` as the port

**Comment by TRusselo** [comment]
Log Details (ERROR)
Logger: hole
Source: /usr/local/lib/python3.9/site-packages/hole/__init__.py:53
First occurred: 10:52:23 PM (1 occurrences)
Last logged: 10:52:23 PM

Can not load data from *hole: pihole:80


TLDR; same result
tried this too
![image](https://user-images.githubusercontent.com/1154815/155071528-ff758e66-0e09-464b-a437-b9ea13b4d15c.png)

**Comment by cliffordwhansen** [comment]
Have you tried unchecking the `Verify SSL certificate` option

**Comment by TRusselo** [comment]
yes tried all on, all off, ect

**Comment by TRusselo** [comment]
ok so its not the documentations... its me! bummer

**Comment by cliffordwhansen** [comment]
Sorry man, but yeah, seems like a network issue between your docker containers.
If you install a terminal addon for HomeAssistant (or connect to the container), you can try pinging the PIHole host from the HA container.
Might even be a firewall rule?

**Comment by TRusselo** [comment]
thanks, ill head to the unraid forum or discord for assistance.
